Makefile

```diff
@@ -91,11 +91,18 @@ localunittest: all
 	go test -timeout 3m -tags "$(BUILDTAGS)" ${TESTFLAGS} -v $(allpackages)
 
 integration: runcimage
-	docker run -e TESTFLAGS -t --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_IMAGE) make localintegration
+	docker run -e TESTFLAGS -ti --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_IMAGE) make localintegration
```

Comment:
You don't really need `-i` here, do you? We removed this to fix CI in non-interactive mode, see https://github.com/opencontainers/runc/pull/1252 .